tutorials/server/setup-zfs-storage-pool/setup-zfs-storage-pool.md

```diff
@@ -0,0 +1,105 @@
+---
+id: setup-zfs-storage-pool
+summary: Create a ZFS storage pool using zpool
+categories: server
+tags: tutorial,storage pool,zfs,file server
+difficulty: 2
+status: Published
+published: 2017-12-05
+author: Aden Padilla <adenpadilla@gmail.com>
+
+---
+
+# Setup ZFS Storage Pool
+
+## Overview
+Duration: 1:00
+
+A ZFS storage pool can be comprised of multiple hard drives. When storage space in a pool is running low, we can simply add extra drives to expand it.
+
+This guide will go through the process of installing ZFS and setting up a storage pool.
+
+### What you'll learn
+- How to install ZFS
+- How to create a storage pool
+
+### What you'll need
+- Ubuntu Server 16.04 LTS
+
+Ready? Let's head over to the next step!
+
+## Installing ZFS
+Duration: 1:00
+
+To install ZFS, run:
+```bash
+sudo apt install zfs
```

Comment:
Have you checked this? I thought the package name was zfsutils-linux

Reply:
it worked for me during the run through but you're right, zfs isn't the package name, it does this redirect thing when you use it.

![image](https://user-images.githubusercontent.com/16405940/33612370-44165152-da0c-11e7-90bb-4a0ce5740307.png)

should i change it?